dffml/source/source.py

```diff
@@ -29,12 +29,45 @@ def __init__(self, parent: "BaseSource") -> None:
     async def update(self, record: Record):
         """
         Updates a record for a source
+
+        Examples
+        --------
+        >>> async def main():
+        ...     async with ExampleSource(records=[]) as testSource:
+        ...         # Open, update, and close
+        ...         async with testSource() as sourceContext:
+        ...             example = Record("one")
+        ...             # ... Update one into our records ...
+        ...             await sourceContext.update(example)
+        ...             # Let's check out our records after calling `record` and `update`.
+        ...             ITER = sourceContext.records()
+        ...             async for ITEM in ITER:
+        ...                 print(ITEM)
+        >>>
+        >>> asyncio.run(main())
+        one
+        Predictions
+        Undetermined
         """
 
     @abc.abstractmethod
     async def records(self) -> AsyncIterator[Record]:
         """
         Returns a list of records retrieved from self.src
+
+        Examples
+        --------
+        >>> async def main():
+        ...     async with ExampleSource(records=[Record("example")]) as testSource:
+        ...         async with testSource() as sourceContext:
+        ...             ITER = sourceContext.records()
+        ...             async for ITEM in ITER:
+        ...                 print(ITEM)
```

Comment:
```suggestion
        ...                 print(record.export())
```

As you can see record printing has become odd. We should probably avoid it and print the exported version.

Reply:
Yes, it had been itching at me. Thank you.